bika/health/content/doctor.py

```diff
@@ -50,5 +66,26 @@ def getARs(self, analysis_state):
         bc = getToolByName(self, 'bika_catalog')
         return [p.getObject() for p in bc(portal_type='AnalysisRequest', getDoctorUID=self.UID())]
 
+    def get_clients(self):
+        # Only show clients to which we have Manage AR rights.
+        mtool = getToolByName(self, 'portal_membership')
+        clientfolder = self.clients
+        clients = []
+        for client in clientfolder.objectValues("Client"):
+            if not mtool.checkPermission(ManageAnalysisRequests, client):
+                continue
+            clients.append([client.UID(), client.Title()])
+        clients.sort(lambda x, y: cmp(x[1].lower(), y[1].lower()))
```

Comment:
Although it is just a matter of style, to sort alphabetically (which I guess is the objective) I find 
```python
clients.sort(key=lambda x: x[1].lower())
```
easier to understand. I don't know, however, which method is more efficient in terms of performance.